ts/packages/ts/src/commands/ExportCommand.ts

```diff
@@ -37,9 +38,22 @@ export class ExportCommand extends Command<ExportCommandReturn> {
     });
   }
 
+  private filterPipe(pipe: EvalCommandReturn) {
+    const { filter } = this.parameters;
+    if (!filter) {
+      return _.pickBy(pipe, ({ context }) => !context.cfgu.hidden);
+    }
+    return _.pickBy(pipe, filter);
+  }
+
+  private mapPipe(pipe: EvalCommandReturn) {
+    return _.mapValues(pipe, (current) => current.result.value);
+  }
```

Comment:
I think you mistakingly left this unused method, I'll remove it for you so we can approve and finalize this PR.